addons/mail/static/src/components/composer/composer.xml

```diff
@@ -48,12 +48,27 @@
                         </t>
                     </div>
                 </t>
+                <t t-if="composer.thread and composer.thread.mass_mailing">
+                    <div class="o_Composer_subject">
+                        <input class="o_Composer_subjectInput" type="text" placeholder="Subject"
+                               t-esc="composer.subjectInputContent"
+                               t-on-focusin="_onFocusinSubject"
+                               t-on-focusout="_onFocusoutSubject"
+                               t-on-keyup="_onKeyupSubject"
+                               t-on-input="_onInputSubject"
+                               t-ref="subject"/>
+
+                    </div>
+                </t>
                 <t t-if="props.hasCurrentPartnerAvatar">
                     <div class="o_Composer_sidebarMain">
                         <img class="o_Composer_currentPartnerAvatar rounded-circle" t-att-src="currentPartnerAvatar" alt=""/>
                     </div>
                 </t>
-                <div class="o_Composer_coreMain" t-att-class="{ 'o-composer-is-compact': props.isCompact }">
+                <div class="o_Composer_coreMain" t-att-class="{
+                    'o-composer-is-compact': props.isCompact,
+                    'o-composer-is-extended': composer.thread and composer.thread.mass_mailing,
+                }">
```

Comment:
`t-att-class` is exception where can use multiline, but then the `t-att-class` itself should be on a new line too, with one more indent for its content and the close tag "alone on its line"

Reply:
Better if we can get rid of `o-composer-is-extended` if that's just for one css rule though, see above comment.